src/main/resources/META-INF/resources/primefaces-extensions/gchart/gchart.js

```diff
@@ -39,6 +39,15 @@ PrimeFaces.widget.ExtGChart = PrimeFaces.widget.BaseWidget.extend({
             });
         });
 
+        // user extension to configure gchart
+        var extender = this.cfg.extender;
+        if (extender) {
+            if (typeof extender === "function") {
```

Comment:
Oh, it's not the name of the function.. it _is_ the function. Can you please clarify that in the attribute description?

Reply:
Sorry, I'm not quite sure how to clarify the attribute description. I used the same description used in primefaces (eg schedule component) or primefaces-extensions (sheet component). It is exactly that: the name of a Javascript function which is called during initialization.

Reply:
Just to give a small code snippet on how to use this:
```
	<script src="https://www.gstatic.com/charts/loader.js"></script>
	<h:outputScript>
		function extendChart() {
			// this = widget
			console.log("extendChart called...");
			this.options.gantt = {
				criticalPathEnabled: true,
				criticalPathStyle: {
					stroke: '#e64a19',
					strokeWidth: 5
				}
			}
		}
	</h:outputScript>
	<h:form>
		<pe:gChart id="gantt" value="#{ganttChartCtrl.chart}" title="GANTT" extender="extendChart">
			<p:ajax listener="#{ganttChartCtrl.onSelect}"/>
		</pe:gChart>
	</h:form>
```